content/departments/engineering/managed-services/telemetry-gateway.md

```diff
@@ -0,0 +1,73 @@
+# Telemetry Gateway infrastructure operations
+
+<!--
+Generated documentation; DO NOT EDIT. Regenerate using this command: 'sg msp operations generate-handbook-pages'
```

Comment:
Nothing to do with this PR, but it might be useful to add some sort of provenance information in the `sg msp` command that shows up here? Like the time the generator last ran, or maybe the commit SHA for any repos that were scraped to gather the data?

e.g.

```
Generated at: 2024-01-11T17:04:27+00:00
sourcegraph/managed-services SHA 57f59a92
```

Reply:
This is a great idea, added to the generator 😁